### ISSUE TITLE
Prevent provisioner to launch with default passwords

### ISSUE THREAD

**Issue by liquidat** [ansible/workshops#449]
##### SUMMARY

Currently we launch the provisioner with default password. While those are only short lived machines anyway, we should make sure that those are run with individual passwords that are not part of the repository where ever possible to increase the security.

For that I suggest: we branch out the play `Perform Checks to make sure this Playbook will complete successfully` in https://github.com/ansible/workshops/blob/devel/provisioner/provision_lab.yml  into its own playbook, and add checks for each password variable in the default vars.

That would also make provision_lab shorter and better to review, which is always a good thing.

This bug report strongly depends on #447.

##### ISSUE TYPE

 - Bug Report

**Comment by IPvSean** [comment]
fixed with https://github.com/ansible/workshops/pull/624

will look like->
```
TASK [make sure we are not using `ansible` as the password] *********************************************************************************************************************************************
fatal: [localhost]: FAILED! => changed=false
  assertion: admin_password != "ansible"
  evaluated_to: false
  msg:
  - admin_password cannot be set to ansible
  - please set a unique password for your workshop
```
